requirements.txt

```diff
@@ -0,0 +1,89 @@
+aiohttp==3.8.4
```

Comment:
These are probably all the dependencies you had in you system's python environment and not just this project's. You'll want to delete this and regenerate them for just the dependencies in the project.

You can create a virtualenv in the root of the directory by running `python -m venv .venv`, reinstall the project dependencies, and then run `pip freeze > requirements.txt` to get just this project's dependencies.